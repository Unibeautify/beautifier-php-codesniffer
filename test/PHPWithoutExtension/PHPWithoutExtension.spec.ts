import * as fs from "fs";
import * as path from "path";

import { newUnibeautify, Beautifier } from "unibeautify";
import beautifier from "../../src";
import { raw } from "../utils";

test(`should successfully beautify php file without extension`, () => {
  const filePath: string = path.resolve(__dirname, `PHPWithoutExtension`);
  const text: string = fs.readFileSync(filePath).toString();
  const unibeautify = newUnibeautify();
  unibeautify.loadBeautifier(beautifier);
  return unibeautify
    .beautify({
      filePath,
      languageName: "PHP",
      options: {
        PHP: {
          PHP_CodeSniffer: {
            prefer_beautifier_config: true,
          },
        } as any,
      },
      text,
    })
    .then(results => {
      expect(raw(results)).toMatchSnapshot();
    });
});
