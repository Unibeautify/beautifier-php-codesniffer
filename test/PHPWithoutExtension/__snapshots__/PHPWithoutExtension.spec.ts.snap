// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should successfully beautify php file without extension 1`] = `
<?php

$var = true;
`;
